src/panels/config/integrations/integration-panels/zha/zha-network-visualization-page.ts

```diff
@@ -266,18 +285,21 @@ export class ZHANetworkVisualizationPage extends LitElement {
 
   private _getLQI(lqi: number): EdgeOptions["color"] {
```

Comment:
To fix the CI: 
```suggestion
  private _getLQI(lqi: number): EdgeOptions {
```

Reply:
Changed. Good catch! It is not 100% correct to use EdgeOptions but I guess it is better than create new type or split this function to two.